ui/src/app/controllers/MainController.js

```diff
@@ -17,13 +17,18 @@
     vm.title = $state.current.data.title;
     vm.showSimpleToast = ToastService.showToast;
     vm.toggleRightSidebar = toggleRightSidebar;
+    vm.logout = logout;
 
     navService
       .loadAllItems()
       .then(function(menuItems) {
         vm.menuItems = [].concat(menuItems);
       });
 
+    function logout() {
```

Comment:
Function hoisting may pollute the global. Use this format
var functionName = function(){
//body...
}